zwave_js_server/const.py

```diff
@@ -10,6 +10,14 @@
 VALUE_UNKNOWN = "unknown"
 
 
+class CommandStatus(str, Enum):
+    """Status of a command sent to zwave-js-server."""
+
+    ACCEPTED = "accepted"
+    QUEUED = "queued"
+    FAILED = "failed"
```

Comment:
We don't seem to use the `FAILED` enum. Do we still need it?

Reply:
We can remove it. I was thinking it would be nice in the future if, for queued messages, we could support an optional callback in the future that returned the result once the message was processed which is why I had added this. But I will remove it for now and we can discuss that separately